docs/docusaurus.config.js

```diff
@@ -71,30 +71,12 @@ const config = {
         respectPrefersColorScheme: false,
       },
       navbar: {
-        style: "dark",
         title: "DBSP",
         logo: {
           alt: "DBSP",
           src: "img/logo.svg",
         },
         items: [
-          {
-            type: "docSidebar",
-            sidebarId: "docsSidebar",
-            position: "right",
-            label: "Get Started",
-          },
-          {
```

Comment:
I think it's good to keep a Link to the REST API as a shortcut in the top navigation. Many users will write programs against that API and will need to access it to look something up.

Reply:
I moved it to the left sidebar. We can reconsider later.